src/components/ui/Hero/HeroContent.tsx

```diff
@@ -0,0 +1,31 @@
+import type { HTMLAttributes } from 'react'
+import React, { forwardRef } from 'react'
+
+export interface HeroContentProps
+  extends Omit<HTMLAttributes<HTMLDivElement>, 'role'> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+}
+
+const HeroContent = forwardRef<HTMLDivElement, HeroContentProps>(
+  function HeroContent(
+    { testId = 'store-hero-content', children, ...otherProps },
+    ref
+  ) {
+    return (
+      <div
+        ref={ref}
+        role="region"
```

Comment:
I'm not sold on the use of `role=region` here... I was wondering if it would make more sense to have `Hero` be a `section` instead of an `article` so we wouldn't need to have this `role=region` here.

Or maybe keep `Hero`'s `article` but replace `role=region` here for a `section`. [An `article` can contain multiple `section`s](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article#:~:text=possibly%20with%20one%20or%20more%20%3Csection%3Es%20within) and [it's recommended to use the correct semantic HTML (`section`) over using ARIA (`role=region`)](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/region_role#:~:text=Using%20the%20%3Csection%3E%20element%20will%20automatically%20communicate%20a%20section%20has%20a%20role%20of%20region%20if%20it%20is%20given%20an%20accessible%20name.%20Developers%20should%20always%20prefer%20using%20the%20correct%20semantic%20HTML%20element%2C%20in%20this%20case%20%3Csection%3E%2C%20over%20using%20ARIA.).